app/src/gui/components/record/form.tsx

```diff
@@ -44,7 +44,11 @@ import {
 } from '../../../context/popup';
 import {selectActiveUser} from '../../../context/slices/authSlice';
 import {store} from '../../../context/store';
-import {percentComplete, requiredFields} from '../../../lib/form-utils';
+import {
```

Comment:
Seeing a problem in here somewhere when I click on a link to a child record, get a spinner because type_cached is being reset to null when the revision_id changes in componentDidUpdate.